src/backendbot/api_routes.py

```
@@ -678,3 +539,399 @@ def get_logs(limit: int = 100) -> list[dict[str, Any]]:
+@router.post("/clean-disk", dependencies=[Depends(get_api_key)])
+def clean_disk():
+    """Limpia archivos temporales y cache del disco."""
+    try:
+        import shutil
+        import os
+        temp_dir = os.environ.get('TEMP', 'C:\\Windows\\Temp')
+        shutil.rmtree(temp_dir, ignore_errors=True)
+        os.makedirs(temp_dir, exist_ok=True)
+        # Desfragmentar (opcional, requiere admin)
+        import subprocess
+        subprocess.run(['defrag', 'C:', '/O'], capture_output=True)
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)